### PR TITLE
Add Behance plugin

### PR DESCRIPTION
**Pull request by azhang496**

### CODE REVIEW

**Comment by kaicataldo** [comment]
:+1:

File: package.json [hunk 1]
```diff
@@ -12,5 +12,11 @@
   "bugs": {
     "url": "https://github.com/behance/eslint-preset-behance/issues"
   },
-  "homepage": "https://github.com/behance/eslint-preset-behance#readme"
+  "homepage": "https://github.com/behance/eslint-preset-behance#readme",
+  "devDependencies": {
+    "eslint": "^3.1.1"
```
Comment (hzoo) [inline package.json]: shouldn't need the devDep since we aren't running tests or anything

Comment (kaicataldo) [inline package.json]: Ohhh yeah, good catch. I'm assuming this was left over from when we thought we would put the rules in the preset itself.

**Comment by hzoo** [comment]
Also this reminds me - I think we can't do this yet because of https://github.com/eslint/eslint/issues/3458

At the moment you have to install dependencies as peerdependencies https://github.com/feross/eslint-config-standard/blob/00fabb06f7622e8762d3a657a7c364993b22d0dd/package.json#L15

**Comment by hzoo** [comment]
👍

**Pull request merged**